src/simularium/RemoteSimulator.ts

```diff
@@ -255,53 +260,52 @@ export class RemoteSimulator implements ISimulator {
         return `wss://${this.serverIp}:${this.serverPort}/`;
     }
 
-    private connectToUriAsync(address): Promise<string> {
-        const connectPromise = new Promise<string>((resolve) => {
-            this.connectToUri(address);
-            resolve("Successfully connected to uri!");
-        });
-
-        return connectPromise;
-    }
-
-    public connectToRemoteServer(address: string): Promise<string> {
+    public connectToRemoteServer(
+        address: string,
+        timeout = 1000
```

Comment:
New parameter `timeout` allows testing with a very short timeout (like 1 instead of 1000 ms), as a way of simulating a slow connection